plugin-annotation/scripts/annotation_manager.java.ejs

```diff
@@ -80,8 +87,62 @@ public class <%- camelize(type) %>Manager extends AnnotationManager<<%- camelize
   @VisibleForTesting
   <%- camelize(type) %>Manager(@NonNull MapView mapView, @NonNull MapboxMap mapboxMap, @NonNull Style style, @NonNull CoreElementProvider<<%- camelize(type) %>Layer> coreElementProvider, @Nullable String belowLayerId, @Nullable GeoJsonOptions geoJsonOptions, DraggableAnnotationController<<%- camelize(type) %>, On<%- camelize(type) %>DragListener> draggableAnnotationController) {
     super(mapView, mapboxMap, style, coreElementProvider, draggableAnnotationController, belowLayerId, geoJsonOptions);
+<% if (type === "symbol") { -%>
+    mapView.addOnStyleImageMissingListener(new ImageLoader(mapView));
   }
 
+  private static class ImageLoader implements MapView.OnStyleImageMissingListener {
+
+    private static final String TAG = "ImageLoader";
+    private WeakReference<MapView> mapViewWeakReference;
+
+    ImageLoader(MapView mapView) {
+      this.mapViewWeakReference = new WeakReference<>(mapView);
+    }
+
+    @Override
+    public void onStyleImageMissing(@NonNull final String id) {
+      if (id.isEmpty()) {
+        return;
+      }
+
+      Logger.v(TAG, String.format("Style image missing %s", id));
```

Comment:
We should decrease the level and change the message or skip this log, otherwise, it's going to pollute the logcat on each load.